test/e2e/specs/fakerp/forceupdate.go

```diff
@@ -0,0 +1,61 @@
+package fakerp
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/openshift-azure/pkg/cluster/updateblob"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+)
+
+var _ = Describe("Force Update E2E tests [ForceUpdate][Fake][LongRunning]", func() {
+	var (
+		cli *azure.Client
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = azure.NewClientFromEnvironment(true)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should be possible for an SRE to force update a cluster", func() {
+		By("Reading the cluster state")
+		before, err := cli.OpenShiftManagedClustersAdmin.Get(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"))
+		Expect(err).NotTo(HaveOccurred())
+		Expect(before).NotTo(BeNil())
+
+		By("Reading the update blob before the force update")
+		ubs, err := updateblob.NewBlobService(cli.BlobStorage)
+		Expect(err).ToNot(HaveOccurred())
+		updateBlobBefore, err := ubs.Read()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(updateBlobBefore).NotTo(BeNil())
+		Expect(len(updateBlobBefore.InstanceHashes)).To(BeEquivalentTo(3)) // one per master instance
+		Expect(len(updateBlobBefore.ScalesetHashes)).To(BeEquivalentTo(2)) // one per worker scaleset
+
+		By("Executing force update on the cluster.")
+		update, err := cli.OpenShiftManagedClustersAdmin.ForceUpdateAndWait(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"))
+		Expect(err).NotTo(HaveOccurred())
+		Expect(update.StatusCode).To(Equal(http.StatusOK))
+		Expect(update).NotTo(BeNil())
+
+		By("Reading the cluster state after the update")
+		after, err := cli.OpenShiftManagedClustersAdmin.Get(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"))
+		Expect(err).NotTo(HaveOccurred())
+		Expect(after).NotTo(BeNil())
+
+		By("Reading the update blob after the force update")
+		updateBlobAfter, err := ubs.Read()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(updateBlobAfter).NotTo(BeNil())
+
+		By("Verifying that the update blob differs before and after the update")
+		Expect(!reflect.DeepEqual(updateBlobBefore, updateBlobAfter)).To(Equal(true))
```

Comment:
Better: `Expect(foo).To(Equal(false))`

Can you look into why you're expecting the blob to actually have changed?